sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwapRfRExamples.cs

```diff
@@ -0,0 +1,195 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using LusidFeatures;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Instruments
+{
+    [TestFixture]
+    public class InterestRateSwapRfRExamples: DemoInstrumentBase
+    {
+        /// <inheritdoc />
+        protected override void CreateAndUpsertMarketDataToLusid(string scope, ModelSelection.ModelEnum model, LusidInstrument instrument)
+        {
+            // The price of a swap is determined by the price of the fixed leg and floating leg.
+            // The price of a floating leg is determined by historic resets rates and projected rates.
+            // In this method, we upsert reset rates.
+            // For LUSID to pick up these quotes, we have added a RIC rule to the recipe (see BuildRecipeRequest in TestDataUtilities.cs) 
+            // The RIC rule has a large quote interval, this means that we can use one reset quote for all the resets.
+            // For accurate pricing, one would want to upsert a quote per reset.
+            InterestRateSwap irs = instrument as InterestRateSwap;
+
+            FloatingLeg floatLeg =
+                (FloatingLeg) irs.Legs.Where(x => x.InstrumentType == LusidInstrument.InstrumentTypeEnum.FloatingLeg).First();
+
+            var indexName = floatLeg.LegDefinition.IndexConvention.IndexName;
+
+            UpsertQuotesResponse upsertResponse;
```

Comment:
Why instantiate here?

Reply:
Was a hangover from a previous iteration of the code - will remove.